src/AutoRest.CSharp/CommandLineOptions.cs

```diff
@@ -28,7 +28,7 @@ internal class CommandLineOptions
         [Option(longName: "server", Required = false, Default = null, HelpText = "Server argument.")]
         public string? Server { get; set; }
 
-        [Option('c', "clear-output-folder", Required = false, Default = false, HelpText = "Clear the output folder before generating code.")]
+        [Option('X', "clear-output-folder", Required = false, Default = false, HelpText = "Clear the output folder before generating code.")]
```

Comment:
Strictly, it is breaking change. But I actually don't have any other ideas except break this.

Reply:
I believe this should be acceptable because it never worked previously

Reply:
should we go with `x` since all the others are lower case?

Reply:
updated.